scripts/get_facet_hierarchy.py

```diff
@@ -1,29 +0,0 @@
-#!/usr/bin/env python
```

Comment:
The functionality of this file was moved to the views.py. Instead of having to run it, and then copy the output into a hardcoded list of facets into the config, the facets will be read from opensearch the first time that they are needed.